src/test/scala/org/broadinstitute/dsde/firecloud/service/NotificationsApiServiceSpec.scala

```diff
@@ -7,9 +7,13 @@ import spray.http.StatusCodes
 import spray.routing.HttpService
 import spray.testkit.ScalatestRouteTest
 
+import scala.concurrent.duration._
+
 class NotificationsApiServiceSpec extends FlatSpec with BeforeAndAfterAll with HttpService with ScalatestRouteTest with NotificationsApiService with Matchers with FireCloudRequestBuilding {
   def actorRefFactory = system
 
+  implicit val routeTestTimeout = RouteTestTimeout(5.seconds)
```

Comment:
looks like most, if not all, other route tests have this 5 second timeout. jenkins seems to be too slow to handle the default 1 second timeout